utils/tenantsConfig.js

```diff
@@ -0,0 +1,8 @@
+import tenantConfig from './../tenant.json';
+export default function getTenantConfig(tenantName){
+    switch(tenantName){
```

Comment:
Ff testing for a tenant key the variable better should be named `tenantKey` here?

Reply:
Resolved this